src/commands/schema/apply.js

```diff
@@ -0,0 +1,77 @@
+const { Command, flags } = require('@oclif/command');
+const fs = require('fs');
+const path = require('path');
+const SchemaSerializer = require('../../serializers/schema');
+const SchemaSender = require('../../services/schema-sender');
+const JobStateChecker = require('../../services/job-state-checker');
+
+class ApplyCommand extends Command {
+  async run() {
+    const logError = this.error.bind(this);
+    const { args, flags: parsedFlags } = this.parse(ApplyCommand);
+    const { projectId } = args;
+    const serializedSchema = this.readSchema();
+    const secret = this.getEnvironmentSecret(parsedFlags);
+
+    const jobId = await new SchemaSender(serializedSchema, secret, logError).perform();
+
+    if (jobId) {
+      await new JobStateChecker('Processing schema', logError).check(jobId, projectId);
+      this.log('Schema successfully sent to forest.');
+    } else {
+      this.log('The schema is the same as before, nothing changed.');
+    }
+
+    return null;
+  }
+
+  readSchema() {
+    const filename = path.resolve('./forestadmin-schema.json');
+
+    if (!fs.existsSync(filename)) {
+      this.error('Cannot find the file "forestadmin-schema.json" in this directory. Please be sure to run this command inside your project directory.', { exit: 1 });
+    }
+
+    let schema;
+    try {
+      schema = JSON.parse(fs.readFileSync(filename, 'utf8'));
+    } catch (error) {
+      this.error(`Invalid json: ${error.message}`, { exit: 1 });
+    }
+
+    return new SchemaSerializer(schema.collections, schema.meta).serialize();
+  }
+
+  getEnvironmentSecret(parsedFlags) {
+    let secret;
+
+    if (parsedFlags.secret) {
+      secret = parsedFlags.secret;
+    } else if (process.env.FOREST_ENV_SECRET) {
+      this.log('Using the forest environment secret found in the environment variable "FOREST_ENV_SECRET"');
+      secret = process.env.FOREST_ENV_SECRET;
+    } else {
+      this.error('Cannot find your forest environment secret in the environment variable "FOREST_ENV_SECRET".\nPlease set the "FOREST_ENV_SECRET" variable or pass the secret in parameter using --secret.', { exit: 2 });
+    }
+
+    return secret;
+  }
+}
+
+ApplyCommand.description = 'Send the forestadmin-schema.json to forest.';
+
+ApplyCommand.args = [
+  {
+    name: 'projectId', required: true, description: 'Forest project ID',
+  },
+];
+
+ApplyCommand.flags = {
+  secret: flags.string({
+    char: 's',
+    description: 'Environment secret of the project',
```

Comment:
FOREST_ENV_SECRET ? By reading this message I'm not sur what you meant. The user does not have to think, we need to make it easy for them

Reply:
```suggestion
    description: 'Environment secret of the project (forest_env_secret)',
```